website/views/options/social/group.jade

```diff
@@ -117,7 +122,7 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
           td
             .popover.static-popover.fade.right.in.wide-popover
               .arrow
-              h3.popover-title {{Members.members[group.leader].profile.name}}
```

Comment:
There was a method of getting the group leader's username already in the file! I think `group.leader.profile.name` is cleaner, but I still can't believe I missed this.